docker/Dockerfile

```diff
@@ -98,14 +98,11 @@ RUN pip install -U pip setuptools
 
 FROM base AS blueoil-base
 # Install requirements
+# We need to install cython and numpy before "pip install -r" for pycocotools
+RUN pip install --upgrade cython numpy
```

Comment:
By this line, we cannot manage which version will install. We should specify the version.

IMO: It is better to add `cython` and `numpy` (version specified) into `requirements.txt` and remove `pycocotools`, add `pip install pycocotools==2.0.0` after `pip install -r /home/blueoil/gpu.requirements.txt` in Dockerfile.

Reply:
Agreed with you. But adding `cython` and `numpy` do not solve dependency problem. I don't know the detail.
After install cython and numpy, `pip install -r` is succeeded.

OK, I'll add the version number.